src/5.0_to_5.1/calculated-measures/migrateSavedCalculatedMeasureContent.ts

```diff
@@ -1,42 +1,31 @@
+import { lt } from "semver";
+
+import { getMigratedSavedCalculatedMeasureContentJSON } from "./getMigratedSavedCalculatedMeasureContentJSON";
+import { getMigratedSavedCalculatedMeasureContentXML } from "./getMigratedSavedCalculatedMeasureContentXML";
+
 /**
  * Transforms the content of a calculated measure created with ActiveUI 5.0, into one that is natively supported by ActivePivot.
+ * Returns its serialized version.
  */
 export const migrateSavedCalculatedMeasureContent = (
   legacyCalculatedMeasureContent: { expression: string; properties: string[] },
   calculatedMeasureName: string,
-): {
-  className: string;
-  additionalProperties: { [propertyName: string]: string };
-  uniqueName: string;
-  expression: string;
-  formatStringExpression?: string;
-} => {
-  const { expression, properties } = legacyCalculatedMeasureContent;
-
-  const formatStringProperty = properties.find((property) =>
-    property.startsWith("FORMAT_STRING"),
-  );
-  // The `FORMAT_STRING` property is a string with the following syntax: "FORMAT_STRING = \\"#,###.##\\"".
-  // Splitting it by " = " returns an array with the property name at index [0] and the value at index [1].
-  const formatStringExpression = formatStringProperty
-    ? formatStringProperty.split(" = ")[1]
-    : undefined;
+  contentServerVersion?: string,
+): string => {
+  if (contentServerVersion && lt(contentServerVersion, "5.11.0")) {
```

Comment:
I believe that we can write `getMigratedSavedCalculatedMeasureContentXML` and `etMigratedSavedCalculatedMeasureContentJSON` in a simpler way by changing their signature to make them accept a map of properties instead of an array:

```
(name: string; expression: string; propertiesMap: { [key: string]: string }) => ...
```
Indeed, they would then both be able to do:
```
const { FORMAT_STRING: formatStringProperty, ...additionalProperties } = propertiesMap;
```

And we could also easily convert the array of properties to a map here before passing it to these 2 functions, by doing:

```
// properties have the shape: "KEY = VALUE"
const propertiesEntries = legacyCalculatedMeasureContent.properties.map(property => property.split(" = "));
const propertiesMap = Object.fromEntries(propertiesEntries);
```